crates/nu-std/std/formats.nu

```diff
@@ -9,12 +9,22 @@
 # These functions help `open` the files with unsupported extensions such as ndjson.
 #
 
-# Convert from ndjson to structured data.
+# Convert from NDJSON to structured data.
 export def "from ndjson" []: string -> any {
     from json --objects
 }
 
-# Convert from jsonl to structured data.
+# Convert from JSONL to structured data.
 export def "from jsonl" []: string -> any {
     from json --objects
 }
+
+# Convert structured data to NDJSON.
+def "to ndjson" []: any -> string {
+    each { to json --raw } | to text
+}
+
+# Convert structured data to JSONL.
+def "to jsonl" []: any -> string {
+    each { to json --raw } | to text
```

Comment:
Does it make sense to define this in terms of `to ndjson`? Or do we want to avoid the small performance hit in the std-lib

Reply:
tbh, i just copied the same idea from the `from ...` commands, i.e. they do not call the other one :yum: